src/demo/demoUtils.js

```diff
@@ -16,7 +16,10 @@ export const getInitSdkOptions = () => {
 
   const language = queryParamToValueString.language === 'customTranslations' ?
     {
-      phrases: { 'welcome.title': 'My custom title' }
+      phrases: { 'welcome.title': 'My custom title' },
+      mobilePhrases: {
+        "capture.driving_licence.back.instructions": "Custom instructions"
+      }
```

Comment:
Just noticed while doing acceptance, but I think this also needs to be added for the `commonLanguages` object map at line 185-193 for Previewer.

Reply:
Good point. Done